src/components/select/select.js

```diff
@@ -1276,7 +1276,7 @@ function SelectProvider($$interimElementProvider) {
      * Calculate the
      */
     function calculateMenuPositions(scope, element, opts) {
-      var 
+      var
```

Comment:
Note how all of the following variable declarations have a comma `,` at the end except the last one, which has a semi-colon. The semi-colon is the stop of the `var` statement basically, so yes, you can do it :-)